website/docs/tstruct.md

```diff
@@ -124,7 +124,7 @@ class ChildTwo < T::Struct
 end
 ```
 
-If it is imperative that inheritance must be used (not composition, either):
+If the code absolutely must use inheritance and cannot use composition, either:
```

Comment:
IMO, adding a "you should" or "you could" before "either" would make it much cleaner.
```suggestion
If the code absolutely must use inheritance and cannot use composition, you could either:
```

Reply:
we generally avoid writing "you" in documentation, but thanks for the suggestion